.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
What is this, please explain ??

Reply:
@PiyushKalyanpy this is the long term nvm version that is enforced